app/decorators/champ_decorator.rb

```diff
@@ -14,6 +14,6 @@ def value
   end
 
   def description_with_links
-    description.gsub(URI.regexp, '<a target="_blank" href="\0">\0</a>').html_safe if description
+    description.gsub(URI.regexp, '<a target="_blank" href="\0">\0</a>') if description
```

Comment:
arf, j'avoue c'est dur a lire ...
Ce changement est en coordination avec celui effectué sur app/views/users/description/champs/_render_list_champs.html.haml qui lui ... convert html_safe to sanitize...